config/config.go

```diff
@@ -68,6 +68,8 @@ const (
 	aggOracleBlockFinalityDeprecated      = "AggOracle.BlockFinality is deprecated, remove it from configuration"
 	l1InfoTreeSyncBlockFinalityDeprecated = "L1InfoTreeSync.BlockFinality is deprecated, remove it from configuration"
 	bridgeL1SyncBlockFinalityDeprecated   = "BridgeL1Sync.BlockFinality is deprecated, remove it from configuration"
+	lastGERSyncDeprecatedHint             = "LastGERSync is deprecated, use L2GERSync instead"
+	lastGERSyncSyncModeDeprecatedHint     = "LastGERSync.SyncMode is deprecated, remove it from configuration"
 )
 
 type DeprecatedFieldsError struct {
```

Comment:
any particular reason we have this?
if we are providing updated config, then maybe completely remove this?
i know its a safe check to keep but not a fan of this. we need to maintain it

Reply:
> if we are providing updated config, then maybe completely remove this?

We are just providing the defaults, but it can be overridden. Although I agree that it can be tedious to maintain it and it will just grow over time, it is a nice way to fail fast in case configuration is invalid in a transparent way.